calisphere/templates/calisphere/related-collections.html

```diff
@@ -66,7 +66,7 @@
                 <img
                 src="{{ thumbnailUrl }}crop/300x300/{{ item.reference_image_md5 }}"
                 class="thumbnail__image"
-                alt="{{ item.title.0|truncatewords:12 }}">
+                alt="">
```

Comment:
In a few cases like this, the thumbnail component's hover text was the same as the image's alt text, throwing a duplicate label error from pa11y. Removing the image's alt text here is OK, since it's already described via the associated hover text.